### PR TITLE
US104432: Include some demo data around started inactive courses with…

### PR DESCRIPTION
**Pull request by njostonehouse**
… sort logic off.

Fixup the course tile component to handle limited organization data.

### CODE REVIEW

File: src/d2l-course-tile.js [hunk 1]
```diff
@@ -615,9 +618,13 @@ Polymer({
 		this.$$('#courseUpdates').setAttribute('aria-hidden', updates === 0 ? 'true' : 'false');
 	},
 	_checkDateBounds: function(organization) {
+		if( !organization.properties ) {
```
Comment (njostonehouse) [inline src/d2l-course-tile.js]: During the demo, this was `undefined`.

File: src/d2l-course-tile.js [hunk 2]
```diff
@@ -615,9 +618,13 @@ Polymer({
...
+			return;
+		}
+
 		var nowDate = Date.now();
-		var endDate = Date.parse(organization.properties.endDate);
-		var startDate = Date.parse(organization.properties.startDate);
+		var endDate = Date.parse(organization.properties.endDate || Number.MAX_VALUE);
+		var startDate = Date.parse(organization.properties.startDate || Number.MIN_VALUE);
```
Comment (edwardg8) [inline src/d2l-course-tile.js]: interesting, I wonder what happens when endDate or startDate is undefined.

Comment (njostonehouse) [reply src/d2l-course-tile.js]: Yeah, I'm not sure. Actually, I wonder if `Date.parse` fails on `MIN_VALUE` than it does on `undefined`. I'm going to revert this part to at least maintain the previous functionality.

Comment (edwardg8) [reply src/d2l-course-tile.js]: yea, i'm not sure if the value can be negative, safer with 0.

**Pull request merged**